botogram/defaults.py

```diff
@@ -78,11 +78,13 @@ def _help_generic_message(self, bot, commands):
         # Show help on commands
         if len(commands) > 0:
             message.append(bot._("<b>This bot supports those commands:</b>"))
-            for name in sorted(commands.keys()):
-                summary = escape_html(commands[name].summary)
+            for command in sorted(commands.values(),
+                                  key=lambda command: command.order):
```

Comment:
This way, the order of the keys if they have the same `order` is random, which is not good (and it's a regression of the current behavior). The best way to fix this, [since sorts are stable in Python](https://docs.python.org/3/howto/sorting.html#sort-stability-and-complex-sorts), is to sort the commands by name before sorting them by order.